SPECS/erlang/erlang.spec

```diff
@@ -19,52 +24,59 @@ erlang programming language
 
 %build
 export ERL_TOP=`pwd`
-./otp_build autoconf
-sh configure --disable-hipe --prefix=%{_prefix}
-
+./configure
 make
 
 %install
 
-make install DESTDIR=%{buildroot}
+%make_install DESTDIR=%{buildroot}
 
 %post
 
 %files
 %defattr(-,root,root)
 %license LICENSE.txt
-%{_bindir}/*
-%{_libdir}/*
-%exclude /usr/src
-%exclude %{_libdir}/debug
+%{usr_local_bin}/ct_run
```

Comment:
Do you know why we suddenly moved to the local `bin` and `lib` directories? I currently don't think it's a big deal but for the sake of consistency, I'm curious, if we can avoid that by restoring the `--prefix=%{_prefix}` switch in `./configure`.

Reply:
I will give this a try and see if there is any impact/differences. /usr/local/bin and /usr/local/lib are simply the default install locations when ./configure is run with no args, hence the switch of install directories. I suspect there isn't much of a functional difference in this case, but having the consistency between versions is a good catch.

Reply:
As noted below by Olivia, changing the `./configure` to instead use `%configure` has resolved the need for the use of the `/usr/local` directories.